src/main/java/org/isf/medicalinventory/gui/InventoryEdit.java

```diff
@@ -2025,6 +2148,306 @@ private boolean checkParameters(String wardCode, String chargeCode, String disch
+			});
+		}
+		return medicalTypeComboBox;
+	}
+	
+    private JRadioButton getAllRadioButton() {
+    	if (radioButtonAll == null) {
+    		radioButtonAll = new JRadioButton(MessageBundle.getMessage("angal.common.all.btn"));
+    		radioButtonAll.setMnemonic(MessageBundle.getMnemonic("angal.common.all.btn.key"));
+    		radioButtonAll.setSelected(true);
+    	}
+    	radioButtonAll.addActionListener(actionEvent -> {
+    		if (radioButtonAll.isSelected()) {
+    			radioOnlyNonZero.setSelected(false);
+				radioWithMovement.setSelected(false);
+			}
+		});
+		return radioButtonAll;
+	}
+    
+    private JRadioButton getMedicalWithNonZeroQuatityRadioButton() {
+    	if (radioOnlyNonZero == null) {
+    		radioOnlyNonZero = new JRadioButton(MessageBundle.getMessage("angal.inventory.medicalwithonlynonzeroqty.btn"));
+    		radioOnlyNonZero.setMnemonic(MessageBundle.getMnemonic("angal.inventory.medicalwithonlynonzeroqty.btn.key"));
+    	}
+    	radioOnlyNonZero.addActionListener(actionEvent -> {
+			if (radioOnlyNonZero.isSelected()) {
+				radioButtonAll.setSelected(false);
+				radioWithMovement.setSelected(false);
+			}
+		});
+		return radioOnlyNonZero;
+	}
+    
+    private JRadioButton getMedicalWithMovementRadioButton() {
+    	if (radioWithMovement == null) {
+    		radioWithMovement = new JRadioButton(MessageBundle.getMessage("angal.inventory.medicalwithmovementonly.btn"));
+    		radioWithMovement.setMnemonic(MessageBundle.getMnemonic("angal.inventory.medicalwithmovementonly.btn.key"));
+    	}
+    	radioWithMovement.addActionListener(actionEvent -> {
+    		if (radioWithMovement.isSelected()) {
+    			radioButtonAll.setSelected(false);
+    			radioOnlyNonZero.setSelected(false);
+			}
+		});
+		return radioWithMovement;
+	}
+    
+    private JButton getCancelButton() {
+    	if (jButtonCancel == null) {
+    		jButtonCancel = new JButton(MessageBundle.getMessage("angal.common.cancel.btn"));
+    		jButtonCancel.setMnemonic(MessageBundle.getMnemonic("angal.common.cancel.btn.key"));
+    	}
+		jButtonCancel.addActionListener(actionEvent -> {
+			frame.dispose();
+		});
+		return jButtonCancel;
+	}
+	private List<MedicalInventoryRow> loadNewInventoryTable(boolean withNonZeroQty, MedicalType medicalTypeSelected) throws OHServiceException {	
+		List<MedicalInventoryRow> inventoryRowsList = getMedicalInventoryRows(null);
+		if (withNonZeroQty) {
+			inventoryRowsList = inventoryRowsList.stream().filter(inv -> inv.getTheoreticQty() > 0).collect(Collectors.toList());
+		}
+		if (medicalTypeSelected != null) {
+			inventoryRowsList = inventoryRowsList.stream().filter(inv -> inv.getMedical().getType().getDescription().equals(medicalTypeSelected.getDescription())).collect(Collectors.toList());
+		}
+		return inventoryRowsList;
+	}
+    
+	private List<MedicalInventoryRow> loadNewInventoryTable(MedicalType medicalTypeSelected, boolean withMovement) throws OHServiceException {	
```

Comment:
I don't think the method parameter, `withMovement` is used in this method and thus can be removed

Reply:
okay